server/mailer/reports/weekly-content.js

```diff
@@ -0,0 +1,135 @@
+const {
+  default: {
+    array: { partition },
+    date: { formatDateTime }
+  }
+} = require('@sandstreamdev/std');
+
+const templateStyles = require('../common/styles');
+const mailTemplate = require('./weekly-template');
+
+const styles = templateStyles({ maxWidth: 1024 });
+
+const tableRow = data => `<tr><td>${data}</td></tr>`;
+const tableHeaderCell = data => `<th style="${styles.thData}">${data}</th>`;
+const tableCell = data => `<td style="${styles.tdData}">${data}</td>`;
+
+const generateDataTable = ({ header, items, showAuthor, host }) => {
+  let content = '';
+  if (header) {
+    content = tableRow(`<h5 style="${styles.h5}">${header}</h5>`);
+  }
+  content += `<tr><td><table style="${styles.tableData}">`;
+  content += '<thead>';
+  content += '<tr>';
+  content += tableHeaderCell('Name');
+  content += tableHeaderCell('Sack (Cohort)');
+  if (showAuthor) {
+    content += tableHeaderCell('Author');
+  }
+  content += tableHeaderCell('Requested at');
+  content += '</tr>';
+  content += '</thead>';
+
+  content += '<tbody>';
+
+  for (let i = 0; i < items.length; i++) {
+    const { author, name, listName, listId, cohortName, requestedAt } = items[
+      i
+    ];
```

Comment:
something wrong with formatting here??
 should be `items[i]` in one line

Reply:
well, that's prettier